src/__test__/actions/asset.actions.test.js

```diff
@@ -161,7 +161,7 @@ describe('Asset Action tests', () => {
         expect(store.getActions()[1].type).toEqual(NEW_ALLOCATION_SUCCESS);
         mock.onGet().reply(200, loadedAsset);
         return store.dispatch(reloadAssetDetail()).then(() => {
-          expect(store.getActions()[3].type).toEqual(LOAD_ASSET_SUCCESS);
+          expect(store.getActions()[2].type).toEqual(LOAD_ASSET_SUCCESS);
```

Comment:
Use `toContainEqual` as you won't need to keep track of the index

Reply:
Fixed @sachungo